packages/babel-node/src/_babel-node.js

```diff
@@ -0,0 +1,163 @@
+import pathIsAbsolute from "path-is-absolute";
+import commander from "commander";
+import Module from "module";
+import { inspect } from "util";
+import path from "path";
+import repl from "repl";
+import { util } from "babel-core";
+import * as babel from "babel-core";
+import vm from "vm";
+import "babel-polyfill";
+import register from "babel-register";
+
+const program = new commander.Command("babel-node");
+
+program.option("-e, --eval [script]", "Evaluate script");
+program.option("-p, --print [code]", "Evaluate script and print result");
+program.option("-o, --only [globs]", "");
+program.option("-i, --ignore [globs]", "");
+program.option("-x, --extensions [extensions]", "List of extensions to hook into [.es6,.js,.es,.jsx]");
+program.option("-w, --plugins [string]", "", util.list);
+program.option("-b, --presets [string]", "", util.list);
+
+const pkg = require("../package.json");
+program.version(pkg.version);
+program.usage("[options] [ -e script | script.js ] [arguments]");
+program.parse(process.argv);
+
+//
```

Comment:
We have the opportunity to remove theses comments (/cc @loganfsmyth). Would you mind removing them?